docs/index.rst

```diff
@@ -792,12 +792,13 @@ solution, Confluent Replicator is also configured with security.
 
 9. Restart the Replicator connector.
 
-10. Observe that the ``connect-replicator`` consumer group has resumed
-   consumption. Notice several things:
+10. Observe that the ``connect-replicator`` consumer group has resumed consumption. Notice several things:
```

Comment:
Might want to use #. instead of hard-coding numbers. It might auto-accommodate your spacing alignment.

Reply:
@tlblessing Yes, 💯 .  But this whole docs page needs to be converted.  Can we approve/ this PR to quickly fix the build, and then in a separate PR address the `#.` ?

Reply:
https://confluentinc.atlassian.net/browse/DEVX-1378

Reply:
sure.